go/enclave/core/rollup.go

```diff
@@ -23,9 +23,10 @@ type Rollup struct {
 // Hash returns the keccak256 hash of b's header.
 // The hash is computed on the first call and cached thereafter.
 func (r *Rollup) Hash() common.L2RootHash {
-	if hash := r.hash.Load(); hash != nil {
-		return hash.(common.L2RootHash)
-	}
+	// Temporarily disabling the caching of the hash because it's causing bugs
```

Comment:
@tudor-malene Add todo to re-enable?

Reply:
@tudor-malene I think you should also briefly say what the bug was, for future reference.